lib/stat_tracker.rb

```diff
@@ -111,10 +111,11 @@ def away_win_percentage
   end
 
   def best_fans
+    diff_percentage = {}
     home_win_percentage.each do |team, percent|
-      home_win_percentage[team] = (percent - away_win_percentage[team])
+      diff_percentage[team] = (percent - away_win_percentage[team])
     end
-    best_fans_team_id = key_with_max_value(home_win_percentage)
+    best_fans_team_id = key_with_max_value(diff_percentage)
     Team.names_by_id[best_fans_team_id]
```

Comment:
I was using each without an accumulator so the `best_fans_team_id` variable was not receiving the correct team id. Adding the accumulator `diff_percentage` takes care of the issue.